test/sse/test_sse.h

```diff
@@ -27,6 +27,14 @@
 #define align1_double double
 #endif
 
+#ifdef __GNUC__
+#define NOINLINE __attribute__((noinline))
+#elif defined(_MSC_VER)
+#define NOINLINE __declspec(noinline)
+#else
+#define NOINLINE
+#endif
```

Comment:
IIUC this code is only ever run via emcc so I think you can just do `#define NOINLINE __attribute__((noinline))`

Reply:
I have run the test_sse files against Visual Studio in the past. (Where I know that VS and Clang have different semantics on some), so this is kind of looking to preserve that possibility to triage through such things.